src/redux/configureStore.js

```diff
@@ -1,16 +1,13 @@
-import { createStore, combineReducers, applyMiddleware } from 'redux';
-import logger from 'redux-logger';
-import booksReducer from './books/books';
-import checkReducer from './categories/categories';
+import { configureStore } from '@reduxjs/toolkit';
+// import logger from 'redux-logger';
+import booksReducer from './books/bookthunk';
+import categoriesReducer from './categories/categories';
 
-const rootReducer = combineReducers({
-  booksReducer,
-  checkReducer,
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+    categories: categoriesReducer,
+  },
 });
```

Comment:
- [ ] Your app is looking great! Everything works, however, I could not find where did you import `thunk` from `redux-thunk` as this is a requirement for this activity:

Requirements:
![Captura de pantalla 2022-06-17 a la(s) 8 56 47 p  m](https://user-images.githubusercontent.com/83717285/174418098-8688b4bb-f769-41f1-ba46-43ac475723d3.png)

It would be great to use `thunk` in this activity. Let me know if I missed something.

Reply:
Hello @smunozmo ! Thanks for your review. I used Redux Toolkit (the latest recommendation from Redux to use since createStore is getting deprecated soon) which has it built-in when you use configureStore. https://redux-toolkit.js.org/api/createAsyncThunk is where it creates the thunk action creators.

Reply:
Hi @myelin0,
You're right, this requirement needs to be updated, thank you for reaching out, I'm glad you got approved and this issue was solved.

Happy coding! 🤓